applications/nrf5340_audio/src/bluetooth/le_audio_bis_gateway.c

```diff
@@ -290,13 +294,25 @@ static int initialize(void)
 		subgroup_params[i].params_count = ARRAY_SIZE(stream_params);
 		subgroup_params[i].params = &stream_params[i];
 		subgroup_params[i].codec = &lc3_preset.codec;
+#if (CONFIG_BT_AUDIO_BROADCAST_IMMEDIATE_FLAG)
+		/* Immediate rendering flag */
+		subgroup_params[i].codec->meta[0].data.type = 0x09;
```

Comment:
Would the type defined in somewhere else so we can avoid the magic number in here?

Reply:
It's coming: https://github.com/zephyrproject-rtos/zephyr/pull/53746